server/build/release.mk

```diff
@@ -224,41 +212,12 @@ package-linux-arm64: package-prep
 package-linux: package-linux-amd64 package-linux-arm64
 
 package-windows: package-prep
-	@# Create needed directories
-	mkdir -p $(DIST_PATH_WIN)/bin
-	mkdir -p $(DIST_PATH_WIN)/logs
-	mkdir -p $(DIST_PATH_WIN)/prepackaged_plugins
-
-	@# Copy binary
-ifeq ($(BUILDER_GOOS_GOARCH),"windows_amd64")
-	cp $(GOBIN)/mattermost.exe $(GOBIN)/mmctl.exe $(DIST_PATH_WIN)/bin # from native bin dir, not cross-compiled
-else
-	cp $(GOBIN)/windows_amd64/mattermost.exe $(GOBIN)/windows_amd64/mmctl.exe $(DIST_PATH_WIN)/bin # from cross-compiled bin dir
-endif
-	@# Prepackage plugins
-	@for plugin_package in $(PLUGIN_PACKAGES) ; do \
-		ARCH="windows-amd64"; \
-		case $$plugin_package in \
-			"mattermost-plugin-calls"*) continue ;; \
-		esac; \
-		cp tmpprepackaged/$$plugin_package-$$ARCH.tar.gz $(DIST_PATH_WIN)/prepackaged_plugins; \
-		cp tmpprepackaged/$$plugin_package-$$ARCH.tar.gz.sig $(DIST_PATH_WIN)/prepackaged_plugins; \
-		HAS_ARCH=`tar -tf $(DIST_PATH_WIN)/prepackaged_plugins/$$plugin_package-$$ARCH.tar.gz | grep -oE "dist/plugin-.*"`; \
-		if [ "$$HAS_ARCH" != "dist/plugin-windows-amd64.exe" ]; then \
-			echo "Contains $$HAS_ARCH in $$plugin_package-$$ARCH.tar.gz but needs dist/plugin-windows-amd64.exe"; \
-			exit 1; \
-		fi; \
-		gpg --verify $(DIST_PATH_WIN)/prepackaged_plugins/$$plugin_package-$$ARCH.tar.gz.sig $(DIST_PATH_WIN)/prepackaged_plugins/$$plugin_package-$$ARCH.tar.gz; \
-		if [ $$? -ne 0 ]; then \
-			echo "Failed to verify $$plugin_package-$$ARCH.tar.gz|$$plugin_package-$$ARCH.tar.gz.sig"; \
-			exit 1; \
-		fi; \
-	done
+	DIST_PATH_GENERIC=$(DIST_PATH_WIN) CURRENT_PACKAGE_ARCH=windows_amd64 MM_BIN_NAME=mattermost.exe MMCTL_BIN_NAME=mmctl.exe $(MAKE) package-general
 	@# Package
 	cd $(DIST_PATH_WIN)/.. && zip -9 -r -q -l ../mattermost-$(BUILD_TYPE_NAME)-windows-amd64.zip mattermost ../mattermost && cd ../..
 	@# Cleanup
 	rm -rf $(DIST_ROOT)/windows
 
-package: package-osx package-linux package-windows
+package: package-linux package-osx package-windows
```

Comment:
Why this change? Is there any difference?

Reply:
> Switched order to fail fast if plugins fail
:)